engine/src/main/java/org/terasology/world/block/regions/BlockRegionComponent.java

```diff
@@ -8,15 +8,19 @@
 import org.terasology.world.block.BlockRegion;
 
 /**
+ *
  */
 public class BlockRegionComponent implements Component {
+    /**
+     * May be null.
+     */
     @Replicate
-    public BlockRegion region = new BlockRegion();
+    public BlockRegion region;
```

Comment:
I think this will have some implications, as it could not be null before.

I haven't double-checked where this component is used/accessed.

Reply:
hmm, so an empty blockmap is null.